Trouble happens at runtime when attempting to link against MPICH2 using pkgconfig, as 'plib' 
is understood to being defined twice.
Signed-off-by: Lucas C. Villa Real <lucasvr@gobolinux.org>

--- mpich2-1.4.1p1/src/packaging/pkgconfig/mpich2-cxx.pc.in.orig	2011-10-20 15:52:48.000000000 -0200
+++ mpich2-1.4.1p1/src/packaging/pkgconfig/mpich2-cxx.pc.in	2011-10-20 15:52:58.000000000 -0200
@@ -4,15 +4,9 @@ exec_prefix=@exec_prefix@
 libdir=@libdir@
 includedir=@includedir@
 
-if test "@NEEDSPLIB@" = "yes" ; then
-   plib=@PMPILIBNAME@
-else
-   plib=
-fi
-
 Name: mpich2
 Description: MPICH2 is an implementation of the Message-Passing Interface (MPI). The goals of MPICH2 are to provide an MPI implementation for important platforms, including clusters, SMPs, and massively parallel processors. It also provides a vehicle for MPI implementation research and for developing new and better parallel programming environments.
 Version: @MPICH2_VERSION@
 Requires:
-Libs: @WRAPPER_LDFLAGS@ -L${libdir} -l${plib} -l@MPILIBNAME@ @WRAPPER_LIBS@
+Libs: @WRAPPER_LDFLAGS@ -L${libdir} -l@MPILIBNAME@ @WRAPPER_LIBS@
 Cxxflags: @WRAPPER_CPPFLAGS@ @WRAPPER_CXXFLAGS@ -I${includedir}
